apps/ledger-live-desktop/src/renderer/components/NewCarousel/index.tsx

```diff
@@ -0,0 +1,67 @@
+import { Icons } from "@ledgerhq/react-ui";
+import useEmblaCarousel, { UseEmblaCarouselType } from "embla-carousel-react";
```

Comment:
Why do we use `embla-carousel-react`? Is there any reason not to stick with `react-transition-group`?

Reply:
`react-transition-group` is an old and unmaintained library whereas `embla-carousel` is a maintained fully fledge customizable carousel. Btw, afaik for this specific thing we weren't using `react-transition-group` but `react-spring`. That being said, I think we shouldn't re-invent the wheel and stick with the most common/used package for basic things such as a carousel. Less thing to maintain on our side.

Reply:
We will introduce a new way to manage the carousel without taking into consideration existing components. We have the Carousel component in our Design System that relies on `react-transition-group` for the slide effect: https://github.com/LedgerHQ/ledger-live/blob/develop/libs/ui/packages/react/src/components/transitions/TransitionSlide.tsx.
Is this component still used? Can we replace it with the new one you created? Otherwise, we will have developers that use the new one and some the other from the DS.
+ Regarding the choice of `embla-carousel` it needs to be discussed because if we all add libraries for specific needs the bundle will explode in size. We need to have a global strategy for how we deal with animations. As we don't only use slides we may benefit from one library to manage our animations such as `react-spring` if possible.
+ We depend on an RC here: `rc19`